rskj-core/src/main/java/co/rsk/net/NodeMessageHandler.java

```diff
@@ -127,7 +126,8 @@ private void tryAddMessage(Peer sender, Message message) {
 
             double score = sender.score(System.currentTimeMillis(), message.getMessageType());
 
-            if (score >= 0 && !this.queue.offer(new MessageTask(sender, message, score))) {
+            boolean notAdded = !this.queue.offer(new MessageTask(sender, message, score));
```

Comment:
Messages should be offered only on non negative scores

Reply:
extracted that part into `addMessage` method